paper-dropdown-menu-light.html

```diff
@@ -474,7 +475,12 @@
          * The content element that is contained by the dropdown menu, if any.
          */
         get contentElement() {
-          return Polymer.dom(this.$.content).getDistributedNodes()[0];
+          var nodes = Polymer.dom(this.$.content).getDistributedNodes();
+          var i = 0;
+          while (nodes[i] && nodes[i].nodeType !== Node.ELEMENT_NODE) {
```

Comment:
Is this still needed? If yes, then can we add a comment because this looks random otherwise